src/test/test_time_of_flight.cxx

```diff
@@ -0,0 +1,500 @@
+
+    test_nonTOF_proj_matrix_sptr.reset(new ProjMatrixByBinUsingRayTracing());
+    dynamic_cast<ProjMatrixByBinUsingRayTracing*>(test_nonTOF_proj_matrix_sptr.get())->set_num_tangential_LORs(1);
+    dynamic_cast<ProjMatrixByBinUsingRayTracing*>(test_nonTOF_proj_matrix_sptr.get())->set_up(test_nonTOF_proj_data_info_sptr, test_discretised_density_sptr);
+
+    // Switch to true in order to export the LORs at files in the current directory
+    test_tof_kernel_application(false);
+}
+
+void
+TOF_Tests::test_tof_proj_data_info()
+{
+    const int correct_tof_mashing_factor = 39;
+    const int num_timing_positions = 9;
+    float correct_width_of_tof_bin = test_scanner_sptr->get_size_of_timing_pos() *
+            test_proj_data_info_sptr->get_tof_mash_factor() * 0.299792458f/2;
+    float correct_timing_locations[num_timing_positions] = {-360.201f/2 + correct_width_of_tof_bin/2, -280.156f/2 + correct_width_of_tof_bin/2,
+                                                            -200.111f/2 + correct_width_of_tof_bin/2, -120.067f/2 + correct_width_of_tof_bin/2,
+                                                            0.0f, 40.022f/2 + correct_width_of_tof_bin/2,
+                                                            120.067f/2 + correct_width_of_tof_bin/2, 200.111f/2 + correct_width_of_tof_bin/2,
+                                                            280.156f/2+ correct_width_of_tof_bin/2};
+
+    check_if_equal(correct_tof_mashing_factor,
+                   test_proj_data_info_sptr->get_tof_mash_factor(), "Different TOF mashing factor.");
+
+    check_if_equal(num_timing_positions,
+                   test_proj_data_info_sptr->get_num_tof_poss(), "Different number of timing positions.");
+
+    for (int timing_num = test_proj_data_info_sptr->get_min_tof_pos_num(), counter = 0;
+         timing_num <= test_proj_data_info_sptr->get_max_tof_pos_num(); ++ timing_num, counter++)
+    {
+        Bin bin(0, 0, 0, 0, timing_num, 1.f);
+
+        check_if_equal(static_cast<double>(correct_width_of_tof_bin),
+                       static_cast<double>(test_proj_data_info_sptr->get_sampling_in_k(bin)), "Error in get_sampling_in_k()");
+        check_if_equal(static_cast<double>(correct_timing_locations[counter]),
+                       static_cast<double>(test_proj_data_info_sptr->get_k(bin)), "Error in get_sampling_in_k()");
+    }
+
+    float total_width = test_proj_data_info_sptr->get_k(Bin(0,0,0,0,test_proj_data_info_sptr->get_max_tof_pos_num(),1.f))
+            - test_proj_data_info_sptr->get_k(Bin(0,0,0,0,test_proj_data_info_sptr->get_min_tof_pos_num(),1.f))
+            + test_proj_data_info_sptr->get_sampling_in_k(Bin(0,0,0,0,0,1.f));
+
+    set_tolerance(static_cast<double>(0.005));
+    check_if_equal(static_cast<double>(total_width), static_cast<double>(test_proj_data_info_sptr->get_coincidence_window_width()),
+                   "Coincidence widths don't match.");
```

Comment:
This puzzles me as `get_coincidence_window_width` is defined in terms of `get_max*` only
https://github.com/UCL/STIR/blob/d88e98cba4e749eb763c61acd2b3cf3c218d687f/src/include/stir/ProjDataInfo.inl#L178-L179
There seems to be a factor 2 wrong in this test (or in `get_coincidence_window_width`)

Reply:
ah no. I misread that. Ignore the previous comment. (I'll delete it)